imap_processing/ultra/tests/unit/test_decom_apid_880.py

```diff
@@ -0,0 +1,80 @@
+import pandas as pd
+import pytest
+
+from imap_processing import decom
+
+
+@pytest.fixture()
+def decom_test_data(ccsds_path, xtce_aux_path):
+    """Read test data from file"""
+    data_packet_list = decom.decom_packets(ccsds_path, xtce_aux_path)
+    return data_packet_list
+
+
+def test_ultra_apid_880_length(decom_test_data):
+    """Test if total packets in data file is correct"""
+    total_packets = 23
+    assert len(decom_test_data) == total_packets
+
+
+def test_enumerated_apid_880(decom_test_data):
+    """Test if enumerated values derived correctly"""
+
+    for packet in decom_test_data:
+
+        if packet.data["SPINPERIODVALID"].raw_value == 0:
```

Comment:
Should these be split up into separate tests? It's hard to tell from looking at the test if all of these if statements are being hit from the packets in the loop.